addons/utm/data/utm_demo.xml

```diff
@@ -1,26 +1,44 @@
-<?xml version="1.0"?>
+<?xml version="1.0" ?>
```

Comment:
Might as well change to `<?xml version="1.0" encoding="utf-8"?>` for consistency.